README.md

```diff
@@ -221,6 +221,31 @@ SUPPORTED_TENSORFLOW_OPS = if_v2([]) + if_not_v2([
     "//tensorflow_recommenders_addons/dynamic_embedding/core:_cuckoo_hashtable_ops.so",
 ]
 ```
+#### GPU Serving TensorFlow models with custom ops
+Including CPU Serving modification, TFRA modification again([`tensorflow_recommenders_addons.bzl`](tensorflow_recommenders_addons/tensorflow_recommenders_addons.bzl)):
+```
+if cuda_srcs:
+    copts = copts + if_cuda(["-DGOOGLE_CUDA=1"])
+    cuda_copts = copts + if_cuda_is_configured([
+        "-x cuda",
+        "-nvcc_options=relaxed-constexpr",
+        "-nvcc_options=ftz=true",
+    ])
+    cuda_deps = deps + if_cuda_is_configured(["//tensorflow_recommenders_addons/dynamic_embedding/core/lib/nvhash:nvhashtable"]) + if_cuda_is_configured([
```

Comment:
https://github.com/tensorflow/recommenders-addons/blob/master/tensorflow_recommenders_addons/dynamic_embedding/core/BUILD#L17
This is necessary？ I have added here for nvhash deps.